src/block/BaseSign.php

```diff
@@ -107,6 +157,32 @@ public function setText(SignText $text) : self{
 		return $this;
 	}
 
+	/**
+	 * Returns a color of Sign text.
+	 */
+	public function getTextColor() : DyeColor{
+		return $this->textColor;
+	}
+
+	/** @return $this */
+	public function setTextColor(DyeColor $color) : self{
+		$this->textColor = $color;
```

Comment:
Simple question: Should this method also change the `BaseSign::$text` property with wrapped sign text like #L138?

Reply:
Is there anything to be gained by doing so?